.github/workflows/pages.yaml

```diff
@@ -0,0 +1,28 @@
+name: GitHub Actions Pages Publish
+on: [push]
+jobs:
+  pages-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - name: Setup node v16
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16' # 16 is default, but lets make sure
+      - name: Install Wrangler2 globally
+        run: npm install -g wrangler@beta
```

Comment:
would you rather encourage they install it into their devDependencies?

Reply:
@threepointone any particular reason? or just to make sure it's specific to the project?

Reply:
well yeah it'll lock the version and won't break automatically
I imagine they'd use it anyway if they want to use functions and stuff
and then optionally can use caching on npm modules so this action runs fast